iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -340,18 +340,30 @@ static LogicalResult setRootConfig(
       workloadPerWorkgroup,
       /*workgroupSize =*/ArrayRef<int64_t>{});
 
+  SmallVector<int64_t> matmulL1TileSizes(clMatmulL1TileSize.begin(),
+                                         clMatmulL1TileSize.end());
+  if (clMatmulL1TileSize.empty()) {
+    Optional<std::string> triple = getTargetTriple(entryPointFn);
```

Comment:
Missed this when `getTargetTriple` was first added. That method should just return the `llvm::TargetTriple` object which can construct from the string attr value. (https://github.com/llvm/llvm-project/blob/491efa7f31cbc39fe0aaad261a840cc5311346fb/llvm/include/llvm/ADT/Triple.h#L278) . After that you can just use `isX86()` method to check here.